app/Http/Middleware/CheckBanned.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Filament\Notifications\Notification;
+
+class CheckBanned
```

Comment:
This is still named "banned"

```suggestion
class CheckUserSuspension
```

Reply:
We've been talking about not suspending the user account, and kicking them out of the panel, but just suspending the servers. So it will probably change

Reply:
Do you want to finish it Charles or is there something you would like me to do for this PR?

Reply:
Let it be for now. We're going to have more discussion about it, I'll finish it from here.